README.md

```diff
@@ -42,7 +42,7 @@ Contributions, issues, and feature requests are welcome!
 
 ## Show your support
 
-Give a ⭐️ if you like this project!
+Give a ⭐️ if you like this project and thank you!
 
 ## Acknowledgments
```

Comment:
- [ ] I think it would be great if you add the acknowledgement for the creator for the original design like this 

> Original design idea by [Cindy Shin in Behance](https://www.behance.net/adagio07).